2021-09-02 15:42:01 [main] [<clinit>] [46] INFO  com.danko.provider.connection.ConnectionFactory - Properties for database have loaded...
2021-09-02 15:42:01 [main] [<init>] [70] INFO  com.danko.provider.connection.ConnectionPool - Connection pool created.
2021-09-02 15:42:01 [main] [executeSelectQuery] [39] ERROR danko.provider.domain.dao.JdbcTemplate - Error...Message: Unknown column 'datetime' in 'field list'
2021-09-02 15:42:47 [main] [<clinit>] [46] INFO  com.danko.provider.connection.ConnectionFactory - Properties for database have loaded...
2021-09-02 15:42:47 [main] [<init>] [70] INFO  com.danko.provider.connection.ConnectionPool - Connection pool created.
